README.md

```diff
@@ -30,13 +30,13 @@ docker run -d -p 3306:3306 -p 9000:9000 --name=memsql memsql/quickstart
 On Mac
 
 ```
-open "http://$(boot2docker ip):9000"
+open "http://$(docker inspect --format '{{ .NetworkSettings.IPAddress }}' memsql):9000"
```

Comment:
++ seems reasonable, maybe just say that MemSQL Ops listens on port 9000 by default and MemSQL listens on port 3306.  The user can do the rest.  :)